presto-docs/src/main/sphinx/sql/grant.rst

```diff
@@ -8,7 +8,7 @@ Synopsis
 .. code-block:: none
 
     GRANT ( privilege [, ...] | ( ALL PRIVILEGES ) )
-    ON [ TABLE ] table_name TO ( user | USER user | ROLE role )
+    ON ( table_name | TABLE table_name | SCHEMA schema_name) TO ( user | USER user | ROLE role )
     [ WITH GRANT OPTION ]
 
 Description
```

Comment:
Mention the semantic of grant privileges to table (in separate commit) and then for schema